pyuvdata/tests/test_miriad.py

```diff
@@ -995,7 +995,7 @@ def test_readWriteReadMiriad_partial_bls(select_kwargs):
 
     # check partial read selections
     full = UVData()
-    full.read(testfile)
+    uvtest.checkWarnings(full.read, [testfile], known_warning='miriad')
```

Comment:
This check warning catch should not be necessary I think. We added a global ignore for the miriad altitude warnings. Unless you are specifically looking to check that the warning is raised in this test. If removing this does cause the warning to re-appear that will be interesting.

Reply:
I think that crept in accidentally, I'll take it out. Thanks!